app/src/main/res/xml/changelog_master.xml

```diff
@@ -1,14 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <changelog xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="MissingDefaultResource">
-    <release version="2023.4 - Phone" versioncode="2">
+    <release version="2023.5 - Phone" versioncode="2">
         <change>Add 2 new Android version sensors</change>
         <change>Add new intent extra types for notification commands</change>
+        <change>Matter improvements and fixes</change>
         <change>Bug fixes, design improvements and dependency updates</change>
     </release>
-    <release version="2023.4 - Wear" versioncode="1">
+    <release version="2023.5 - Wear" versioncode="1">
         <change>Entity State Complication: Support long text type, add new option to show or hide friendly name</change>
         <change>Add 2 new Android version sensors</change>
         <change>Bug fixes, design improvements and dependency updates</change>
     </release>
+    <release version="2023.5 - Automotive" versioncode="1">

```

Comment:
We need to make the `versioncode` unique for each release otherwise it will only show the latest one in the changelog.

Reply:
Fixed